src/domUpdates.js

```diff
@@ -178,17 +152,21 @@ const isTagActive = event => event.target.closest(".tag-card")?.querySelector(".
 const removeActiveFromTag = event => event.target.closest(".tag-card").querySelector(".tag-image-bg").classList.remove("active-bg");
 const addActiveToTag = event => event.target.closest(".tag-card").querySelector(".tag-image-bg").classList.add("active-bg")
 
-const makeTagActive = (event) => {  
+const renderActiveTag = (event) => {  
   if (isTagActive(event)) {
```

Comment:
Can we rename this function to start with a verb? Like `checkTagActiveStatus` or `checkIfActive` instead of having the function name be a question?

Reply:
done